lnwallet/commitment.go

```diff
@@ -663,6 +663,36 @@ func CreateCommitTx(chanType channeldb.ChannelType,
 	return commitTx, nil
 }
 
+// CoopCloseBalance returns the final balances that should be used to create
+// the cooperative close tx, given the channel type and transaction fee.
+func CoopCloseBalance(chanType channeldb.ChannelType, isInitiator bool,
```

Comment:
While testing this feature, I found this pre-existing incorrect log message. Maybe fix quickly in this PR?

`2020-08-24 09:01:51.744 [INF] PEER: Local close channel request delivered to peer: [2 98 92 136 50 75 95 159 151 32 77 82 54 241 125 189 123 161 99 149 32 230 55 0 253 76 249 205 20 250 75 75 209]`

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/4558/commits/929cb61dfa712d05db53408dd3d12c84352ebcb9